client/pdo/client/controller/commands/send.py

```diff
@@ -44,10 +46,21 @@ def send_to_contract(state, save_file, enclave, message, quiet=False, wait=False
         raise Exception('unable to load the contract')
 
     # ---------- set up the enclave service ----------
-    try :
-        enclave_client = get_enclave_service(state, enclave)
-    except Exception as e :
-        raise Exception('unable to connect to enclave service; {0}'.format(str(e)))
+    enclave_names = state.get(['Service', 'EnclaveServiceNames'], [])
+    if len(enclave_names) > 0: #use the database to get the list of enclaves for the contract
+        logger.info('Using eservice database to look up service URL for the contract enclave')
+        try:
+            eservice_to_use = random.choice(state.get(['Service', 'EnclaveServiceNames']))
+            db = ServiceDB_Manager(service_type='eservice', file_name=state.get(['Service', 'EnclaveServiceDatabaseFile']))
+            enclave_client = db.get_serviceclient_by_name(eservice_to_use)
```

Comment:
same as above. servicedb manager should be created & initialized on start. pdo-shell commands should add & remove entries over time.